src/test/run-fail/explicit-panic-msg.rs

```diff
@@ -15,5 +15,6 @@
 fn main() {
     let mut a = 1;
     if 1 == 1 { a = 2; }
-    panic!(format!("woooo{}", "o"));
+    let msg = format!("woooo{}", "o");
+    panic!(msg);
```

Comment:
Because otherwise I get stability warnings saying that the internals of float are unstable.

I assume this is due to a bug in either my macro, the expander, or in the way we lint stability via the chain of spans associated with the expansion.  I was/am not sure if this is a deal-breaker for this change; maybe I will have to track down that bug first (and presumably let this wait until 1.x to land.)